src/app/MessageDef/AttributeDatas.cpp

```diff
@@ -37,13 +37,13 @@ using namespace chip::TLV;
 namespace chip {
 namespace app {
 #if CHIP_CONFIG_IM_ENABLE_SCHEMA_CHECK
-CHIP_ERROR AttributeDataList::Parser::CheckSchemaValidity() const
+CHIP_ERROR AttributeDatas::Parser::CheckSchemaValidity() const
```

Comment:
nit: 'Data' is already a plural of datum, AttributeDatas -> AttributeData

Reply:
Done, rename to AttribtueDataIBs. thanks